context.go

```diff
@@ -689,9 +690,9 @@ loop:
 	return deps, errs
 }
 
-type FileHandler func(*parser.File)
+type FileVisitor func(*parser.File)
```

Comment:
Handler makes more sense than Visitor to me, Visitor implies walking something that is already parsed to me.  The implementation above is also still called handleOneFile.

Reply:
done